measures/2018/measures-data.json

```diff
@@ -1407,7 +1407,7 @@
     "nqfEMeasureId": null,
     "nqfId": "0421",
     "measureId": "128",
-    "description": "Percentage of patients aged 18 years and older with a BMI documented during the current encounter or during the previous twelve months AND with a BMI outside of normal parameters, a follow-up plan is documented during the encounter or during the previous twelve months of the current encounter\r\n\r\nNormal Parameters:\r\n\r\nAge 18 years and older BMI � 18.5 and < 25 kg/m2",
+    "description": "Percentage of patients aged 18 years and older with a BMI documented during the current encounter or during the previous twelve months AND with a BMI outside of normal parameters, a follow-up plan is documented during the encounter or during the previous twelve months of the current encounter\r\r\r\rNormal Parameters:\r\r\r\rAge 18 years and older BMI >= 18.5 and < 25 kg/m2",
```

Comment:
i looked into this and couldn't figure out why it's changed. `\r\n` is the windows carriage return and `\r\r` is the os x carriage return but i don't know why that's the only character that's changing... i don't think it should have an effect though?

Reply:
oh interesting... so it's because the amount of whitespace in the source csv doubled :( sarah looked into it and the fields where the whitespace has doubled aren't shown in the shopping cart ui so there's no difference for the user.